source/renderer/zettlr-body.js

```diff
@@ -150,17 +149,40 @@ class ZettlrBody {
    * necessary adjustments.
    */
   configChange () {
+    // On config change, change the theme according to the settings
     let newTheme = global.config.get('display.theme')
-    // Check if we really need to replace the style to prevent ugly flickering
-    // when replacing the same theme with the same.
-    if (this._currentTheme === newTheme) return
 
-    // On config change, change the theme according to the settings
-    let href = $('link#theme-css').attr('href')
-    href = href.replace(/bielefeld|berlin|frankfurt|karl-marx-stadt|bordeaux/, newTheme)
-    $('link#theme-css').attr('href', href)
-    this._renderer.getEditor().refresh()
-    this._currentTheme = newTheme
+    let berlin = require('./../common/assets/less/theme-berlin/theme-main.less')
+    let bielefeld = require('./../common/assets/less/theme-bielefeld/theme-main.less')
+    let frankfurt = require('./../common/assets/less/theme-frankfurt/theme-main.less')
+    let karlMarxStadt = require('./../common/assets/less/theme-karl-marx-stadt/theme-main.less')
+    let bordeaux = require('./../common/assets/less/theme-bordeaux/theme-main.less')
+
+    let themeToUse
+    switch (newTheme) {
+      case 'berlin':
+        themeToUse = berlin
+        break
+      case 'bielefeld':
+        themeToUse = bielefeld
+        break
+      case 'frankfurt':
+        themeToUse = frankfurt
+        break
+      case 'karl-marx-stadt':
+        themeToUse = karlMarxStadt
+        break
+      case 'bordeaux':
+        themeToUse = bordeaux
+        break
+    }
+
+    // Unload all themes except the new one
+    [ berlin, bielefeld, frankfurt, karlMarxStadt, bordeaux ]
+      .filter(theme => theme !== themeToUse)
+      .forEach(theme => theme.unuse())
+    // Load the new theme
+    themeToUse.use()
```

Comment:
Nice find! Is this a webpack built-in function?

Reply:
> ⚠️ Behavior is undefined when unuse is called more often than use. Don't do that.

-- https://github.com/webpack-contrib/style-loader#lazystyletag

Have you experienced any problem so far?

Reply:
No, so far everything works. I added a comment that points to this warning as a reminder if we experience problems in the future.